### PR TITLE
add toggle to cypress tests, off by default

### PR DESCRIPTION
**Pull request by stewartie4**

### CODE REVIEW

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM 🎉, can be merged with corresponding PR in gosdk that sends true for run_frontend_tests

**Comment by Kishan-Dhakan** [comment]
@cnlangzi please review as well, thanks

**Comment by stewartie4** [comment]
do we have a full list of repos that require these tests? is it just gosdk?

File: run-system-tests/action.yml [hunk 1]
```diff
@@ -279,7 +291,8 @@ runs:
         (./zwallet version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZWallet Version Info:/") || true
         (./zbox version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZBox Version Info:/") || true
 
-    - name: "Set up wasm tests"
+    - name: "Set up frontend tests"
+      if:  ${{ env.RUN_FRONTEND_TESTS == 'true' }}
```
Comment (cnlangzi) [inline run-system-tests/action.yml:295]: it should be better to check if gosdk is changed

`if: ${{ inputs.custom_go_sdk_version }} != "" && ${{ inputs.custom_go_sdk_version }} != "NONE"`

Comment (stewartie4) [reply run-system-tests/action.yml:295]: I disagree, this adds unnecessary complexity and could cause gaps in coverage, for example if gosdk was merged to staging with a bug
Easier and safer to always run for a given repo such as godsk imho where we can be sure they'll always run

Comment (stewartie4) [reply run-system-tests/action.yml:295]: to compromise, I added both checks 
@cnlangzi 

if someone overrides gosdk on any repo, frontend tests will run

**Pull request merged**